opentelemetry-sdk/src/logs/growable_array.rs

```diff
@@ -0,0 +1,370 @@
+/// The default max capacity for the stack portion of `GrowableArray`.
```

Comment:
This is pretty generic and we may use it in other signals right, would it make sense for us to move it out of `logs` and put it in it's own mod?

Reply:
I initially thought we would move this code to the global space only if needed, but I think it's more visible and usable there. Done.